hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_indexer.py

```diff
@@ -72,7 +72,9 @@ def merge_pdq_files(
         accumulator[hash] = hash_row
     else:
         # Add new privacy group to existing row
-        accumulator[hash][1]["privacy_groups"].append(meta_data["privacy_groups"][0])
+        accumulator[hash][1]["privacy_groups"].update(meta_data["privacy_groups"])
+        # Add new labels from privacy group to existing row
```

Comment:
complete nit, but from a naming convention, for now, at this point here, this is still te data and so here it's tags not labels, so I'd change Add new labels... to Add new tags...

Reply:
it's in a superposition, schrodinger metadata if you will